deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

```diff
@@ -154,7 +154,7 @@ spec:
             - "--metricspath=/metrics"
             - "--healthzport=9681"
             - "--healthzpath=/healthz"
-            - "--polltime=60s"
+            - "--polltime=15s"
```

Comment:
Just because it "looks long" is not a good reason. Maybe you can look for other projects that do something like this and see what time they default to?

Reply:
Yes, I should have been clear before, got inspired by:
1. https://github.com/kubernetes-csi/livenessprobe/blob/bba64df584a52d98ddf1904b7f8ec20a2828257c/deployment/kubernetes/livenessprobe-sidecar.yaml#L21
2. https://github.com/kubernetes-csi/livenessprobe#usage
    https://github.com/kubernetes-csi/livenessprobe#command-line-options
   https://github.com/kubernetes-csi/livenessprobe/blob/master/deployment/kubernetes/livenessprobe-sidecar.yaml#L21
3. https://github.com/openshift/csi-livenessprobe#usage
   https://github.com/openshift/csi-livenessprobe#other-recognized-arguments
   https://github.com/openshift/csi-livenessprobe/blob/master/deployment/kubernetes/livenessprobe-sidecar.yaml#L21